app/main.py

```diff
@@ -1,8 +1,40 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(
+            self, comfort_class: int, clean_mark: int, brand: str
+    ) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self, distance_from_city_center: float, clean_power: int,
+                 average_rating: float, count_of_ratings: int) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, cars: list) -> float:
+        income_list = []
+        for car in cars:
+            if car.clean_mark < self.clean_power:
+                income_list.append(self.calculate_washing_price(car))
+                self.wash_single_car(car)
+        return sum(income_list)
+
+    def calculate_washing_price(self, car: Car) -> float:
+        price = car.comfort_class * (self.clean_power - car.clean_mark) * \
+            (self.average_rating / self.distance_from_city_center)
```

Comment:
There is a potential division by zero error in the `calculate_washing_price` method if `distance_from_city_center` is zero. Consider adding a check to handle this case to prevent runtime errors.